src/de/jost_net/JVerein/gui/control/FilterControl.java

```diff
@@ -363,9 +365,13 @@ public DialogInput getEigenschaftenAuswahl() throws RemoteException
       }
       try
       {
+        String s = stt.nextToken();
+        String prefix = "+";
+        if (s.substring(s.length()-1).equalsIgnoreCase("2"))
```

Comment:
Ich würde die Zustände in ein enum packen damit der Code lesbarer ist, mit "2" kann man sonst wenig anfangen

Reply:
mache ich